tests/integration/test_merge_tree_blob_storage/configs/config.d/storage_conf.xml

```diff
@@ -0,0 +1,20 @@
+<clickhouse>
+    <storage_configuration>
+        <disks>
+            <blob_storage_disk>
+                <type>blob_storage</type>
+                <endpoint>?</endpoint>
```

Comment:
Why `?` is used for endpoint?
Did you try to put something like here `http://azurite1:10000/`?

>  I run my tests locally with pytest* (not in the docker with ./runner, as I have this NETWORK_ERROR there) 

btw in our CI tests are run with runner, not with pytest. So need to make it work locally with runner.

Reply:
I modified the Disk registration code on the side and didn't use this endpoint for this development phase, eventually it should be something like `http://azurite1:10000/`. I am still figuring out how to connect different pieces and make them work, I have these mentioned issues with connecting different containers. I am also still working out the `NETWORK_ERROR` problem, but I don't have a clear suspect.

Reply:
> I modified the Disk registration code on the side and didn't use this endpoint for this development phase, eventually it should be something like http://azurite1:10000/

Ok, but it could be more convenient if you added some endpoint (at least this one ` http://azurite1:10000/`, it looks correct) to config so it is used when disk is registered and some test in CI started. So it would be easier to understand by CI logs what errors do you get and to checkout to you branch and also try to check how to make it work.

Reply:
Can you please fix build (cmake issues)? Because it does not build now as is. It is easier for me to checkout to your branch and check myself what is going on with your tests. I have added really a lot of integration tests myself for external engines/storages including those which had no support before, so your case should have little difference.